### ISSUE TITLE
[Feature Request] Weather icon animated

### ISSUE THREAD

**Issue by 2ndwonder** [benphelps/homepage#164]
**Is your feature request related to a service? Please describe.**
Is it possible to make the weather icon animated?

**Additional context**

<img width="587" alt="image" src="https://user-images.githubusercontent.com/24813161/190324239-c0f0fd74-05d4-45f5-92a8-2ac7136e104f.png">

**Comment by benphelps** [comment]
I've avoided this namely because animated icon packages (like Skycons) have poor support for High DPI displays, have a limited set of conditions, lack night/day variations and lack theming support.

If you can find a collection of icons or package that provides all of these, I'd be open to adding it as an option.

**Comment by 2ndwonder** [comment]
> I've avoided this namely because animated icon packages (like Skycons) have poor support for High DPI displays, have a limited set of conditions, lack night/day variations and lack theming support.
> 
> If you can find a collection of icons or package that provides all of these, I'd be open to adding it as an option.

https://icons8.com/icon/set/weather/ios--animated
https://shareae.com/after-effects-project/elements/35760-videohive-weather-icons-29564206.html (it's a ae template. maybe you can convert this?)

**Comment by benphelps** [comment]
Creating an icon pack is far outside the scope of the project or anything I'd be willing to do.

Even if Icons8 would work, it's not a free or opensource project, which would be required.

**Comment by 2ndwonder** [comment]
Perhaps I can do it. You need gif extension i suppose?

**Comment by benphelps** [comment]
No, it would need to be animated SVGs (or another method of presenting scalable vector graphics, canvas, css) for proper High DPI and theming support.  Gifs are a fixed size and fixed color (with limited color pallet support), it wouldn't work at all.

If this was easy, it would already be a feature.
